db/migrate/20191221151801_add_is_active_to_company.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsActiveToCompany < ActiveRecord::Migration[6.0]
+  def change
+    add_column :companies, :is_active, :boolean, default: true
```

Comment:
I would vote for not using `is_` as a prefix for boolean columns. Just for the sake of consistency with already existing boolean columns in the database. What do you guys think about it?